### ISSUE TITLE
Qt5 build wrongly configured since commit 2f7820a?

### ISSUE THREAD

**Issue by softrabbit** [LMMS/lmms#2107]
I tried my hand at building with Qt5 and didn't get too far, as the compiler couldn't find the Qt header files. Git bisect points at 2f7820a6d90bfdbd603c5c5a6617bf5fa81b9bd8 being the point where this bug was introduced. 

CMake 3.0.2 on Ubuntu 15.04, if that makes any difference. And master branch of course.

**Comment by softrabbit** [comment]
And it seems to me like shortcircuiting this if clause so the "else" part runs fixes this problem: https://github.com/LMMS/lmms/blob/master/src/CMakeLists.txt#L68 

@Lukas-W, any thoughts?

**Comment by midi-pascal** [comment]
This looks to me as if the operands in this "if" should be "AND" instead of "OR"...

**Comment by softrabbit** [comment]
> This looks to me as if the operands in this "if" should be "AND" instead of "OR"...

Not quite that simple. More like ```major > 2 or ( major == 2 and ( minor > 8 or (minor == 8 and patch > 7 ) ) )```. But luckily, there's only ever been one CMake version (2.4.8) released that would hit that bug :smiley:

**Comment by Lukas-W** [comment]
Yeah… That version comparison is stupid. I should have used CMake's [builtin version comparison](http://www.cmake.org/cmake/help/v2.8.12/cmake.html#command:if), didn't know about it back then.

```cmake
if(<variable|string> VERSION_LESS <variable|string>)
if(<variable|string> VERSION_EQUAL <variable|string>)
```

Could you try using that? If that doesn't fix the issue, let me know and I'll try to build against Qt5 myself.

**Comment by Lukas-W** [comment]
Fixed epr aaaa11c
